docs/getting-started/launching-one-node/docker.md

```diff
@@ -18,23 +18,41 @@ To build a docker image, the following are the prerequisites which should be ins
 
 ### Build
 
-To build the docer image, the following steps need to be executed:
+To build the docker image, the following steps need to be executed:
 
   1. To clone the repository, first, create required directories using the command `mkdir -p github.com/hyperledger-labs`
   2. Change the current working directory to the above created folder by issing the command `cd github.com/hyperledger-labs`
   3. Clone this repository with `git clone https://github.com/hyperledger-labs/orion-server`
   4. Change the current working directory to the repository root directory by issuing `cd orion-server`
-  5. To build the docker images, issue the command `make docker` which will create a docker image named `orion-server`
+  5. Optional. Generate all cryptographic materials. For more info see [crypto-materials](crypto-materials).
+  6. To build the docker images, issue the command `make docker` which will create a docker image named `orion-server`
```

Comment:
docker image or images? are there several?

Reply:
Image, for sure